src/rules/whitespaceRule.ts

```diff
@@ -306,7 +320,16 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 }
         }
     });
-
+    function matchPostblockPreKeyword(node: ts.Node): boolean {
```

Comment:
Consider moving this function out of the `walk` closure. It doesn't use any variables from that scope, so it doesn't need to be here.

Reply:
Done